app/helpers/application_helper.rb

```diff
@@ -34,7 +34,8 @@ def user_context
     {
       admin: current_user ? current_user.role == 1 : false,
       allowedFeatures: current_user && current_user.allowed_feature_list,
-      maxSamplesBulkDownload: get_app_config(AppConfig::MAX_SAMPLES_BULK_DOWNLOAD),
+      maxSamplesBulkDownload: get_app_config(AppConfig::MAX_SAMPLES_BULK_DOWNLOAD).to_i,
```

Comment:
I dont think these should be parameters of this context, since it is a user specific context.
I believe we should create an AppContext for this... wdyt?